src/main/java/uk/gov/ons/census/action/model/entity/ActionRule.java

```diff
@@ -34,7 +27,6 @@
 
   @Column private Boolean hasTriggered;
 
-  @Type(type = "jsonb")
-  @Column(columnDefinition = "jsonb", nullable = false)
-  private Map<String, List<String>> classifiers;
+  @Column(nullable = false, length = 100000)
```

Comment:
100k is big but how did you arrive at that number? Any reasoning based on the maximum number of LSOAs we're likely to classify on based on the rehearsal?

Reply:
why not use a `TEXT` column type?